usaspending_api/agency/v2/views/budget_function.py

```diff
@@ -0,0 +1,94 @@
+from django.db.models import Q, Sum
+from rest_framework.request import Request
+from rest_framework.response import Response
+from typing import Any
+from usaspending_api.agency.v2.views.agency_base import AgencyBase, ListMixin
+from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.common.helpers.generic_helper import get_simple_pagination_metadata
+from usaspending_api.financial_activities.models import FinancialAccountsByProgramActivityObjectClass
+
+
+class BudgetFunctionList(ListMixin, AgencyBase):
+    """
+    Obtain the count of budget functions for a specific agency in a single
+    fiscal year based on whether or not that budget function has ever
+    been submitted in File B.
+    """
+
+    endpoint_doc = "usaspending_api/api_contracts/contracts/v2/agency/toptier_code/budget_function.md"
+
+    def validate_request(self):
```

Comment:
Just out of curiosity..... what's this function?  I don't see it being used anywhere.

Reply:
honestly? no idea. removed it.